Tetris Clone C++/main.cpp

```diff
@@ -87,10 +88,14 @@ int main(int argc, char const *argv[]) {
                         canInstantDrop = false;
                     }
                     else if (event.key.code == sf::Keyboard::D) {
-                        piece.rotateClockwise();
+                        if (piece.rotateClockwise()) {
+
```

Comment:
i'm guessing the compiler complained if you just called a method that returned a bool and did nothing with that bool.